components/list/list.css

```diff
@@ -0,0 +1,70 @@
+/* @import url('../../components/root.css'); */
+
+@import url('../../components/framer.css');
+
+
+.list-bg{
+    background-color: var(--text-col-white);
+    border-radius: 0.313rem;
+}
+
+.text-deocration-none{
+    text-decoration: none;
+}
+.list-style-none{
+    list-style: none;
+}
+
+.list-inline{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.list-square{
+    list-style-type: square;
+    margin-bottom: 0.75rem;
+}
+
+.list-circle{
+    list-style-type: circle;
+    margin-bottom: 0.75rem;
+
+}
+
+.list-disc{
+    list-style-type: disc;
+    margin-bottom: 0.75rem;
+
+}
+
+
+/*for ordered list*/
+
+.list-number{
+    list-style-type:decimal;
+}
+
+.list-lower-lphabet{
+    list-style-type: lower-alpha;
+}
+
+.list-capital-alphabet{
+    list-style-type:upper-alpha;
+}
+
+.list-lower-roman{
+    list-style-type: lower-roman;
+}
+
+/* Image section of notification*/
+
+.notification-container{
+    padding: 1rem 0.5rem 1rem 0.5rem;
+    background-color: var(--text-col-white);
+}
+.notification-img{
+    width: 9.375rem;
+    height: 100%;
+    border-radius: 0.313rem;
+}
```

Comment:
You can give values in px here

Reply:
use 3px instead of 0.313rem. anyways px values are too small so better keep it as integers rather than floating number.